bugbug/test_scheduling_features.py

```diff
@@ -108,3 +108,18 @@ def __call__(self, test_job, **kwargs):
                 "failures_past_56_pushes_in_components"
             ],
         }
+
+class arch(object):
+    def __call__(self, test_job, **kwargs):
+        archs = []
+        architectures = ("64", "32", "x86", "i386", "arm", "aarch64")
```

Comment:
32, x86 and i386 should all be considered 32 bit

Reply:
`aarch64` is a 64 bit implementation of ARM. Do you want me to classify this as ARM or 64 bit?

Reply:
It should be a separate one (Aarch64 or ARM64).